src/main/java/gui/controllers/RoomController.java

```diff
@@ -163,7 +163,7 @@ public void fillProgress(final int stage) {
     /**
      * Update the activity Label.
      */
-    private void updateActivity() {
+    public void updateActivity() {
```

Comment:
waarom is dit naar public gegaan? Als het alleen voor tests is kan je het net zo goed protected (of misschien zelfs package private) maken